locales/en/unsubscribe.ftl

```diff
@@ -17,3 +17,4 @@ unsubscribe-success-cta = Sign in to { -brand-monitor }
 # Error warning
 
 unsubscription-failed = Unsubscribe failed. <try_again_link>Try again.</try_again_link>
```

Comment:
Is this string still used somewhere? If not, it should be removed from the file.